packages/lib/server/checkBookingLimits.ts

```diff
@@ -86,7 +86,7 @@ export async function checkLimit({
       }
 
       throw new HttpError({
-        message: `Booking limit of ${limitingNumber}/${filter} reached for this eventType`,
+        message: `Booking limit of ${limitingNumber}/${filter} reached for this Event Type`,
```

Comment:
This should be translatable - this is meant for the FE but as it is meant for display it needs a translation string. We should name this error so we can use the name as a translation key (I think that's the easiest approach)

Reply:
How would we handle passing in the limitingNumber and filter to the FE error?

Reply:
For now let's keep it simple, we can come up with a more global strategy in the future as we fix similar HttpError's